website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
```

Comment:
The only way to let the popover always be visible, is to set the trigger to "click",  I could add it to this pullrequest